request.d.ts

```diff
@@ -0,0 +1,6 @@
+import { PinejsClientCore } from './core'
+import * as request from 'request'
```

Comment:
Thanks @pimterry!  And I didn't have @types/request because I thought request had them built in, I was tricked by vscode magic though which auto found/used the @types definitions :(